### PR TITLE
Packaging meeting minutes, 2018-02-21

### PR DESCRIPTION
**Pull request by graeme-a-stewart**
Finally had time to finish these off. @drbenmorgan , @ktf can you check the minutes are good?

### CODE REVIEW

**Comment by graeme-a-stewart** [comment]
If no one can look at this today, I think I will accept it...

File: organization/_posts/2018-02-21-packaging.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    -   Git for all sources, even third party
+    -   Bash for all build recipes, main point is allowing contributions
+        from non-experts
+    -   YAML for metadata
+    -   Conventions over templates: conventions on environment that
+        recipe runs in.
+    -   Tarballs as file formats, rpms/dpkgs in separate step by FPM
+    -   Dependency Management:
+        -   No central "config" - derived only from set of dependencies
+            (Graeme: back propagation? Yes)
```
Comment (ktf) [inline organization/_posts/2018-02-21-packaging.md]: We actually do a topological sort to decide the build order, but the list of edges is created via back propagation.

**Comment by ktf** [comment]
Minor comment. It can go in, as far as I am concerned.

**Comment by graeme-a-stewart** [comment]
Thanks - I added the little clarification anyway

**Pull request merged**